src/Command/Anonymization/AnonymizeCommand.php

```diff
@@ -40,81 +101,258 @@ protected function configure(): void
+                'During anonymization, when not set, a single UPDATE statement ' .
+                'will be issued per table, when set, every target will issue its ' .
+                'own UPDATE statement.'
             )
             ->addOption(
                 'target',
                 't',
                 InputOption::VALUE_IS_ARRAY | InputOption::VALUE_REQUIRED,
-                'Only anonymise the given column or table targets.',
-                [],
+                'Only anonymize the given column or table targets.',
             )
             ->addOption(
                 'exclude',
                 'x',
                 InputOption::VALUE_IS_ARRAY | InputOption::VALUE_REQUIRED,
-                'Exclude given targets.'
-            )
-            ->addOption(
-                'force',
-                null,
-                InputOption::VALUE_NONE,
-                'Do not ask for confirmation before restoring database'
+                'Exclude given targets from anonymization.'
             )
         ;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    protected function initialize(InputInterface $input, OutputInterface $output): void
+    {
+        $this->io = new SymfonyStyle($input, $output);
+
+        $this->io->warning([
+            'Note that this command should only be used in a secured environment. Following GDPR best ' .
+            'practices, sensitive data should never transit on an unsecured environment.',
+            'Learn how to use this command in a global GDPR-friendly workflow ' .
+            'reading the DbToolsBundle documentation: ',
+            '> https://dbtoolsbundle.readthedocs.io/en/stable/anonymization/workflow.html',
+        ]);
+
+
+        $this->doAnonymizeCurrentDatabase = !!$input->getOption('current-database');
+
+        if ($input->getOption('no-restore')) {
+            $this->doBackupAndRestoreInitial= false;
+        }
+
+        if (!$input->isInteractive()) {
+            $this->io->warning("--no-interaction is set, no confirmation will be asked.");
+        }
+
+        $this->connectionName = $input->getOption('connection') ?? $this->connectionName;
+        $this->backupFilename = $input->getArgument('filename');
+
+        // Anonymization options
+        $this->excludedTargets = $input->getOption('exclude');
+        $this->onlyTargets = $input->getOption('target');
+        $this->atOnce = !$input->getOption('split-per-column');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function interact(InputInterface $input, OutputInterface $output): void
+    {
+        if (!$this->backupFilename && !$this->doAnonymizeCurrentDatabase) {
+            $this->doCancel = true;
+
+            $this->io->caution([
+                'You should either provide a backup file or use the --current-database option.',
+                'For more informations, Launch this command with --help.'
```

Comment:
```suggestion
                'For more information, launch this command with --help.'
```

"information" is uncountable :wink: